sdk/storage/azure-storage-blob-cryptography/README.md

```diff
@@ -0,0 +1,147 @@
+# Azure Storage Blobs Cryptography client library for Java
+
+Azure Blob storage is Microsoft's object storage solution for the cloud. Blob
+storage is optimized for storing massive amounts of unstructured data.
+Unstructured data is data that does not adhere to a particular data model or
+definition, such as text or binary data.
+
+## Getting started
+
+### Prerequisites
+
+-  Java Development Kit (JDK) with version 8 or above
+- [Azure Subscription][azure_subscription]
+- [Create Storage Account][storage_account]
+
+### Adding the package to your product
+
+```xml
+<dependency>
+  <groupId>com.azure</groupId>
+  <artifactId>azure-storage-blob</artifactId>
+  <version>12.0.0-preview.3</version>
+</dependency>
+```
+
+### Default HTTP Client
```

Comment:
We should have Netty as the default Http client. See https://github.com/Azure/azure-sdk-for-java/issues/5511

This section should be updated accordingly. Refer README for blobs.